### PR TITLE
completes 6-get-author_id-comments

### PR DESCRIPTION
**Pull request by ilterr**

### CODE REVIEW

File: db/models.js [hunk 1]
```diff
@@ -37,3 +37,17 @@ exports.selectArticles = () => {
...
+exports.selectComments = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`,
+      [article_id]
+    )
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "No comments found" });
+      }
```
Comment (nc-ali-abdallah) [inline db/models.js:50]: This shouldn't be an error it should return the empty array. and we are going to need to implement a different way to know if the article we are request exists (either a new checkExists, or using a previous model we have to check an article by ID.)

**Review by nc-ali-abdallah — COMMENTED** [review]
Awesome stuff testing wise.
I'd like to see one more test 200 - if the article is valid but there is no comments we should return an empty array.

Remember to remove console logs when they are no longer needed.

Nice job on this task.
